invenio_i18n/cli.py

```diff
@@ -34,11 +35,11 @@
 
 
 def source_translation_files(input_directory):
```

Comment:
@utnapischtim creating missing language folders in this function may a bit misleading the user. This function creates missing folders for all packages registered in webpack entrypoint but unified json may not include all packages. We may move this logic to `distribute_js_translations` function.

Reply:
We decide to move missing folder creation code after getting target file path in zoom meeting.